src/OpenTelemetry.Exporter.Prometheus.HttpListener/Internal/PrometheusHeadersParser.cs

```diff
@@ -7,7 +7,7 @@ internal static class PrometheusHeadersParser
 {
     private const string OpenMetricsMediaType = "application/openmetrics-text";
 
-    internal static bool AcceptsOpenMetrics(string contentType)
+    internal static bool AcceptsOpenMetrics(string? contentType)
```

Comment:
Can you please double check if this is needed?
I looked at the callers and I don't see anyone passing a null into this method.

Reply:
`var header` [here](https://github.com/open-telemetry/opentelemetry-dotnet/blob/82b1fbd760eacc4bac2e4f7cf754a36f911e6d77/src/OpenTelemetry.Exporter.Prometheus.AspNetCore/PrometheusExporterMiddleware.cs#L110) is actually `string?`. Seems to be a [quirk of enumerating a StringValues struct](https://learn.microsoft.com/dotnet/api/microsoft.extensions.primitives.stringvalues.enumerator.current?view=net-8.0).